genesis/src/genesis_accounts.rs

```diff
@@ -535,13 +535,43 @@ fn add_stakes(
         .sum::<u64>()
 }
 
-pub const VALIDATOR_INFOS: &[ValidatorInfo] = &[ValidatorInfo {
-    name: "aurel@ethereum.ro",
-    node: "GeZ5PrJi9muVCJiJAaFBNGoCEdxGEqTp7L2BmT2WTTy1",
-    vote: "7ZdRx2EBYoRuPfyeoNbuHodMUXcAQRcC37MUw3kP6akn",
-    node_sol: 500.0,
-    commission: 0,
-}];
+pub const VALIDATOR_INFOS: &[ValidatorInfo] = &[
+    ValidatorInfo {
+        name: "01Node",
+        node: "5n8KCdzqtvTnhkvCrFR7errH6ZUp11kL97r2awXkfzFe",
+        vote: "4uYMbY5Ae5ZSRNxQ3RWVyXS9rzW7E3AMZYHuUEotxu6K",
+        node_sol: 500.0,
+        commission: 0,
+    },
+    ValidatorInfo {
+        name: "Bison Trails",
+        node: "7suRNpX7bJsXphHJtBv4ZsLjJZ1dTGeX256pLqJZdEAm",
+        vote: "DfirEZ9Up1xbE7sQji9UwtcRGe5uCcRqQtnaGpha5KNY",
+        node_sol: 500.0,
+        commission: 0,
+    },
+    ValidatorInfo {
+        name: "CyberFund",
```

Comment:
we're not using "name" for anything right now except to uniquely identify a line in the records, to keep the copy-pasta audit-able, but using this column of names will result in non-unique entries...

Reply:
I don't think it should, a validator shouldn't have multiple keys in this list even though the spreadsheet structure currently implies that's possible.

Reply:
I mean: the choice of company for "name" will make it harder to verify that each validator_info entry has the right keys, because the name will match multiple rows in the source sheet

Reply:
One company should only have a single entry at the moment though

Reply:
that's not currently true, though...

Reply:
Yes it is.  Each validator only provided one set of id/vote keys

Reply:
when I consult the validator signup sheet I see multiple validators per company